src/App.js

```diff
@@ -1,24 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import { Navigate, Route, Routes } from "react-router";
+import "./App.css";
+import MainPage from "./page/MainPage";
+import AddItemPage from "./page/AddItemPage";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Routes>
+        <Route path="/" element={<Navigate to="/items" />} />
```

Comment:
여기는 다른 이유가 있는게 아니라면 Naviate 없이 그냥 MainPage 를 넣어도 되지 않을까용..?

```suggestion
        <Route path="/" element={<MainPage />} />
```

Reply:
"중고마켓 페이지 주소는 “/items” 입니다." 라는 요구사항을 보았습니다.
 그래서 처음 렌더링 되었을때, 주소가  /items 이고 그에 해당하는 페이지를 띄우는 것으로 이해했습니다..;;